design-system-docs/src/_data/component_arguments.yml

```diff
@@ -9,3 +9,22 @@ targeted_content:
     description: Optional, sets the heading level of the title. Defaults to 2
   - argument: close_button
     description: Optional, include close button at the end of content. Defaults to true
+radio_group:
+  - argument: name
+    description: Required, field name
+  - argument: label
+    description: Required, the text for the label associated with the input
+  - argument: options
+    description: Optional, a hash with one or more of the following values
+  - argument: hint
```

Comment:
This is probably a question for a future PR but I'm not sure how to indent the arguments that are part of the options, in the old docs it is done with '&rarr; hint' but this throws an error here